src/test/java/com/redhat/cloud/policies/app/NonApplicationRootPathTest.java

```diff
@@ -0,0 +1,39 @@
+package com.redhat.cloud.policies.app;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.http.ContentType.JSON;
+import static io.restassured.http.ContentType.TEXT;
+
+@QuarkusTest
+public class NonApplicationRootPathTest {
```

Comment:
I just noticed that RestAssured follows `301` redirections which is not something we want in this class.

I'm testing a fix in policies-engine. I'll commit here in a few minutes if it works.

Reply:
Fixed.